PL2/PL2-core/src/test/java/nl/tudelft/pl2016gr2/core/algorithms/FilterSnipsTest.java

```diff
@@ -1,51 +1,12 @@
 package nl.tudelft.pl2016gr2.core.algorithms;
 
-import nl.tudelft.pl2016gr2.model.Node;
-import nl.tudelft.pl2016gr2.model.OriginalGraph;
 
 import org.junit.Test;
 
 public class FilterSnipsTest {
 
   @Test
   public void testMakeSnip() {
-    OriginalGraph graph = new OriginalGraph();
```

Comment:
I tried to avoid merge conflicts with pull request #52. This deletion will probably cause one (or two) merge conflicts, but deleting the file might cause more merge conflicts (I am not really sure about this though).

> Pull request #52 (Test/filter-snips) adds tests to this test class